### ISSUE TITLE
Issues with uncompress when padding is single digits

### ISSUE THREAD

**Issue by herronelou** [rsgalloway/pyseq#55]
I've had an interesting bug today where I was trying to uncompress a sequence based on a string with single digit padding (%d), and uncompress kept returning None.

```
>>> import pyseq
>>> sequence = pyseq.get_sequences(['file1.jpg', 'file2.jpg'])[0]
>>> uncompress_sequence = pyseq.uncompress(sequence.format('%h%p%t %r'), fmt='%h%p%t %r')
>>> print uncompress_sequence
None
```

I gave it a try with the code in the issue 43 branch, but it doesn't work either.
I'll try to see if it's an easy fix, and if so I'll do a PR.

**Comment by herronelou** [comment]
I should have looked before I posted, it was a single character change. 
Making a PR now.

**Comment by rsgalloway** [comment]
merged into 1.0.0 branch
